packages/@aws-cdk/custom-resources/test/provider-framework/mocks.ts

```diff
@@ -134,3 +138,25 @@ export async function startExecutionMock(req: AWS.StepFunctions.StartExecutionIn
     startDate: new Date(),
   };
 }
+
+export async function getFunctionMock(_req: AWS.Lambda.GetFunctionRequest): Promise<AWS.Lambda.GetFunctionResponse> {
+  getFunctionInvokeCount++;
+  if (getFunctionInvokeCount === 0) {
+    return {
+      Configuration: {
+        State: 'Inactive',
+      },
```

Comment:
```suggestion
      Configuration: {
        State: 'Inactive',
        LastUpdateStatus: 'Failed',
      },
```

Reply:
Why do we need to check `LastUpdateStatus`? The current function state is enough information, no?